### PR TITLE
Added simple manifest for WDI import

### PR DESCRIPTION
**Pull request by cbari123**

### CODE REVIEW

**Review by jehangiramjad — COMMENTED** [review]
Thanks! Just one question about the TMCF file reference

File: scripts/world_bank/wdi/download_indicators/manifest.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            ],
+            "provenance_url": "https://databank.worldbank.org/source/world-development-indicators",
+            "provenance_description": "World Bank dataset containing various statistics for countries.",
+            "scripts": [
+                "wdi_download_indicators.py"
+            ],
+            "import_inputs": [
+                {
+                    "cleaned_csv": "cleaned_wdi.csv"
+                }
```
[the remark this answers is not in the document]
Comment (cbari123) [reply scripts/world_bank/wdi/download_indicators/manifest.json:16]: For now we don't need the TCMF file, but might add in the future

[the remark this answers is not in the document]
Comment (jehangiramjad) [reply scripts/world_bank/wdi/download_indicators/manifest.json:16]: ok great!

**Review by jehangiramjad — APPROVED** [review]
Thanks

**Comment by jehangiramjad** [comment]
Closing this because a new one was created which is now merged.

**Pull request closed without merging**